src/user/UserIdentityProviderLinks.tsx

```diff
@@ -94,7 +94,7 @@ export const UserIdentityProviderLinks = () => {
       <Link
         to={toIdentityProvider({
           realm,
-          // providerId: idp.identityProvider!,
+          providerId: idp.identityProvider,
```

Comment:
If the `providerId` field is made required in the route you can use the non-null assertion operator to force this value to be interpreted as defined.
```suggestion
          providerId: idp.identityProvider!,
```